src/index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const app = express();
+const PORT = 3000;
+
+const { specs } = require('./swagger');
+const swaggerUi = require('swagger-ui-express');
+
+
+app.get('/', (req, res) => res.send('Home Page'));
+
+app.get('/about', (req, res) => {
+  res.json({ message: 'This is About Page' });
+});
+
+app.use('/healthcheck', require('./routes/healthchecker'));
```

Comment:
Probably not very important, but in HW requirements this endpoint with hyphen - `GET /health-check`.

Reply:
yes, thank you. I`ve changed it)